packages/core/core/src/PackagerRunner.js

```diff
@@ -495,7 +505,14 @@ export default class PackagerRunner {
     };
 
     for (let optimizer of optimizers) {
+      let measurement;
       try {
+        measurement = applicationProfiler.createMeasurement(optimizer.name, {
```

Comment:
Similarly, should this be gated or otherwise made to avoid the object allocations?